data/json/items/armor/ballistic_armor.json

```
@@ -642,6 +625,149 @@
     "looks_like": "esbi_plate",
     "material": [ "damaged_ceramic" ]
   },
+  {
+    "id": "ballistic_vest_heavy",
+    "type": "ARMOR",
+    "category": "armor",
+    "//": "Based on US Army IOTV. Weight from size M base piece plus lower back protector, data from: https://www.moore.army.mil/tenant/lrc/content/pdf/IOTV%20GEN%20III%20TM%2010-8470-210-10.pdf ",
+    "name": { "str": "heavy ballistic vest" },
+    "description": "Heavy ballistic armor with specialized pockets on the front, back, and sides for armored plates.  Includes additional protection for the lower back and attachment points for optional shoulder, groin, and neck protection. ",
```

```suggestion
    "description": "A heavy ballistic armor used by the US Army, known as the IOTV.  It has pockets on the front, back, and sides for armor plates.  Includes additional protection for the lower back and attachment points for optional shoulder, groin, and neck protection. ",
```
This description is a bit more clear.